__tests__/__fixtures__/output/node/axios/full.js

```diff
@@ -1,15 +1,18 @@
-var axios = require("axios").default;
+const axios = require("axios").default;
+const { URLSearchParams } = require('url');
+const encodedParams = new URLSearchParams();
 
-var options = {
+encodedParams.set('foo', 'bar');
+
+const options = {
   method: 'POST',
-  url: 'https://httpbin.org/anything',
-  params: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
```

Comment:
Made this change because query params were getting to HTTPBin as the following:

```js
  args: { baz: 'abc', 'foo[]': [ 'bar', 'baz' ], key: 'value' },
```

With this fix:

```js
  args: { baz: 'abc', foo: [ 'bar', 'baz' ], key: 'value' },
```

Reply:
interesting, so `foo[]` is an old way people used to identify that multiple identical parameters should be treated as an array. I haven't seen it for quite some time though. In fact it doesn't really even seem to be supported as an OAS style. 

I'm going to poke around axios a bit and see why they're still using this pattern by default.

Reply:
Looks like you can use `URLSearchParams`: https://stackoverflow.com/questions/42898009/multiple-fields-with-same-key-in-query-params-axios-request

Reply:
To get this working right with `URLSearchParams` we'll need to run through arrays and add them in separately:

```
const params = new URLSearchParams();
params.append('foo', 'bar');
params.append('foo', 'baz');
params.append('baz', 'abc');
params.append('key', 'value');
```

I'm going to leave it with having query params in the URL because it'll be less noisy.